### PR TITLE
Fix for IPV6 discrepancy

### PR DESCRIPTION
**Pull request by shrsr**

### CODE REVIEW

File: aci/resource_aci_fvsubnet.go [hunk 1]
```diff
@@ -132,20 +132,18 @@ func getRemoteSubnet(client *client.Client, dn string) (*models.Subnet, error) {
 
 func setSubnetAttributes(fvSubnet *models.Subnet, d *schema.ResourceData) (*schema.ResourceData, error) {
 	dn := d.Id()
-	d.SetId(fvSubnet.DistinguishedName)
+	ip := d.Get("ip").(string)
+	d.Set("ip", ip)
```
Comment (lhercot) [inline aci/resource_aci_fvsubnet.go:136]: We should compare ip with fvSubnetMap["ip"] first to make sure it is equivalent then do the d.Set

Something like: https://www.techieindoor.com/go-how-to-compare-ip-addresses-in-go/

File: aci/resource_aci_fvsubnet.go [hunk 2]
```diff
@@ -132,20 +132,18 @@ func getRemoteSubnet(client *client.Client, dn string) (*models.Subnet, error) {
...
 	d.Set("description", fvSubnet.Description)
 	if dn != fvSubnet.DistinguishedName {
-		d.Set("parent_dn", "")
+		d.SetId(dn)
```
Comment (lhercot) [inline aci/resource_aci_fvsubnet.go:139]: This does not seems correct. Why are we setting it to dn if it is not identical?

**Review by lhercot — CHANGES_REQUESTED** [review]

File: aci/resource_aci_fvsubnet.go (hunk 1 above)
Comment (shrsr) [reply aci/resource_aci_fvsubnet.go]: ok

**Review by lhercot — APPROVED** [review]
LGTM

**Pull request merged**